packages/handlersjs-http/lib/handlers/http-cors-request.handler.ts

```diff
@@ -105,12 +92,12 @@ export class HttpCorsRequestHandler extends HttpHandler {
 
             ... response.headers,
             ... allowOrigin && ({
-              ... (allowOrigin !== '*') && { 'Vary': 'Origin' },
-              'Access-Control-Allow-Origin': allowOrigin,
-              'Access-Control-Allow-Methods': (allowMethods ?? routeMethods ?? allMethods).join(', '),
-              ... (allowHeadersOrRequested) && { 'Access-Control-Allow-Headers': allowHeadersOrRequested },
-              ... (credentials) && { 'Access-Control-Allow-Credentials': 'true' },
-              'Access-Control-Max-Age': (maxAge ?? -1).toString(),
+              ... (allowOrigin !== '*') && { 'vary': 'origin' },
+              'access-control-allow-origin': allowOrigin,
+              'access-control-allow-methods': (allowMethods ?? routeMethods ?? allMethods).join(', '),
+              ... (allowHeadersOrRequested) && { 'access-control-allow-headers': allowHeadersOrRequested },
+              ... (credentials) && { 'access-control-allow-credentials': 'true' },
+              'access-control-max-age': (maxAge ?? -1).toString(),
```

Comment:
Why did you change the case?

Reply:
![image](https://user-images.githubusercontent.com/43341653/135036511-e634b317-8fa2-4103-a5be-e85cb1df9148.png)